### ISSUE TITLE
Maven?

### ISSUE THREAD

**Issue by CH-EricLundberg** [3mcloud/GFC-Grouper-Foundation-Classes#5]
Is there a Maven repo for this?  Looking at implementing DRG codes using the https://www.cms.gov/Medicare/Medicare-Fee-for-Service-Payment/AcuteInpatientPPS/MS-DRG-Classifications-and-Software code and it requires this software, being able to load it as a standard dependency seems like it would be a win.

**Comment by mfunaro** [comment]
Currently we do not post this to Maven central or other public maven repository. You can take the jar in the releases and install it locally using maven or upload to a private maven repo you have access to as a workaround for now. 

There were no plans to provide this via maven central or otherwise but I will bring this up for discussion internally.

**Comment by CH-EricLundberg** [comment]
Fair enough, just wanted to make sure I wasn't missing anything.  Would love if cms.gov would publish to maven as well, but so it goes. :)

**Comment by shahafabileah** [comment]
Plus one on this request.  We can set up our own maven repo as a workaround, but it would be one-step easier for adoption if this was already available in a common maven repo.
